src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -802,6 +813,66 @@ public static void TestingGetStringInvalidUTF8(byte[] dataUtf8)
             }
         }
 
+        [Theory]
+        [MemberData(nameof(GetCommentTestData))]
+        public static void TestingGetComment(string jsonData, string expected)
+        {
+            byte[] dataUtf8 = Encoding.UTF8.GetBytes(jsonData);
+            var state = new JsonReaderState(options: new JsonReaderOptions { CommentHandling = JsonCommentHandling.Allow });
+            var reader = new Utf8JsonReader(dataUtf8, isFinalBlock: true, state);
+            bool commentFound = false;
+            bool startObjectFound = false;
+            bool endObjectFound = false;
+            while (reader.Read())
+            {
+                switch (reader.TokenType)
+                {
+                    case JsonTokenType.StartObject:
+                        startObjectFound = true;
+                        break;
+                    case JsonTokenType.EndObject:
+                        endObjectFound = true;
+                        break;
+                    case JsonTokenType.Comment:
+                        commentFound = true;
+                        Assert.Equal(expected, reader.GetComment());
+                        break;
+                    default:
+                        Assert.True(false);
+                        break;
+                }
+            }
+            Assert.True(startObjectFound);
+            Assert.True(commentFound);
+            Assert.True(endObjectFound);
+        }
+
+        [Theory]
+        [MemberData(nameof(GetCommentUnescapeData))]
+        public static void TestGetCommentUnescape(string jsonData, string expected)
+        {
+            byte[] dataUtf8 = Encoding.UTF8.GetBytes(jsonData);
+            var state = new JsonReaderState(options: new JsonReaderOptions { CommentHandling = JsonCommentHandling.Allow });
+            var reader = new Utf8JsonReader(dataUtf8, isFinalBlock: true, state);
+            bool commentFound = false;
+            while (reader.Read())
```

Comment:
This test could be simplified since it is only a single token within the json.

```C#
Assert.True(reader.Read());
Assert.Equal(JsonTokenType.Comment, reader.TokenType);

string comment = reader.GetComment();
Assert.Equal(expected, comment);
Assert.NotEqual(Regex.Unescape(expected), comment);
```